src/Components/ImagesTable/ImageBuildErrorDetails.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Flex } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+
+const useGetErrorReason = (err) => {
+    if (!err || !err.reason) {
+        return 'An unknown error occured';
+    }
+
+    if (err.details && err.details.reason) {
+        return err.details.reason;
+    }
+
+    return err.reason;
+};
+
+const FailureIcon = () => (
+    <Flex className="pf-u-align-items-baseline pf-m-nowrap">
+        <div className="pf-u-mr-sm">
+            <ExclamationCircleIcon className="error" />
+        </div>
+        <strong className='pf-u-danger-color-100'>Image build failed</strong>
+    </Flex>
+);
+
+const ErrorDetails = ({ status }) => {
+    if (!status || status.status !== 'failure') {
+        return <React.Fragment />;
+    }
+
+    const reason = useGetErrorReason(status.error);
+
+    return (
+        <div className='pf-u-mt-sm'>
+            <strong>Status</strong>
+            <FailureIcon />
+            <p className='pf-u-danger-color-100 pf-u-w-33-on-md'>
+                { reason }
+            </p>
+        </div>
+    );
+};
+
+ErrorDetails.propTypes = {
```

Comment:
This is not a critical comment and I am not questioning it... but just for my own knowledge, why do we need this?

Reply:
Think it's required by the linter to validate the type of props being passed to the component. Think it's just for type safety.